src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingService.kt

```diff
@@ -441,6 +446,21 @@ class BehandlingService(
             BehandlingMapper.tilDomeneBehandling(opprettTilbakekrevingRequest, fagsystem, fagsak, ansvarligsaksbehandler)
         behandlingRepository.insert(behandling)
 
+        val manuelleBrevmottakere = brevmottakere.map { brevmottaker ->
+            ManuellBrevmottaker(
+                behandlingId = behandling.id,
+                type = MottakerType.valueOf(brevmottaker.type.name),
+                adresselinje1 = brevmottaker.manuellAdresseInfo.adresselinje1,
+                adresselinje2 = brevmottaker.manuellAdresseInfo.adresselinje2,
+                postnummer = brevmottaker.manuellAdresseInfo.postnummer,
+                poststed = brevmottaker.manuellAdresseInfo.poststed,
+                landkode = brevmottaker.manuellAdresseInfo.landkode,
+                navn = brevmottaker.navn,
+                vergetype = brevmottaker.vergetype
+            )
```

Comment:
Med den nyeste versjonen av familie-felles kan vi også mappe ident og orgnr:
```suggestion
            ManuellBrevmottaker(
                behandlingId = behandling.id,
                type = MottakerType.valueOf(brevmottaker.type.name),
                ident = brevmottaker.personIdent,
                orgNr = brevmottaker.organisasjonsnummer,
                adresselinje1 = brevmottaker.manuellAdresseInfo.adresselinje1,
                adresselinje2 = brevmottaker.manuellAdresseInfo.adresselinje2,
                postnummer = brevmottaker.manuellAdresseInfo.postnummer,
                poststed = brevmottaker.manuellAdresseInfo.poststed,
                landkode = brevmottaker.manuellAdresseInfo.landkode,
                navn = brevmottaker.navn,
                vergetype = brevmottaker.vergetype
            )
```